.github/workflows/code-scan.yml

```
@@ -348,18 +353,33 @@ jobs:
           cache-read-only: true  # Force read only, even in main
           add-job-summary: 'on-failure'
 
+      - name: "Validate input CodeQL queries"
```

Flytt dette til en egen valideringsjobb på toppen av workflow. Fail early som de sier :)